.github/workflows/ci.yml

```
@@ -17,9 +17,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04, ubuntu-20.04]
-        elixir_version: [1.15, 1.16]
-        otp_version: [24, 25, 26]
+        os: [ubuntu-22.04]
+        elixir_version: [1.15, 1.16, 1.17]
+        otp_version: [ 25, 26, 27]
```

nit: Looks a bit strange and might need cleanup?
```suggestion
        otp_version: [25, 26, 27]
```